CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Versions
 ========
 
+2019-07-DEV
```

Comment:
Sorry, just a last review : Please directly update `2019-06-DEV` below into `2019-07-DEV` since your contribution will be in the same release than these ones.

Thank you !